update.sh

```diff
@@ -107,7 +110,12 @@ ENUMS_WITH_DEFAULTS=${ENUMS_WITH_DEFAULTS:1}
 GATEWAY_API_ENUMS=${ENUMS_WITH_DEFAULTS} cargo xtask gen_enum_defaults >> gateway-api/src/apis/experimental/enum_defaults.rs
 echo "mod enum_defaults;" >> gateway-api/src/apis/experimental/mod.rs
 
-GATEWAY_CONDITION_CONSTANTS=${GATEWAY_CONDITION_CONSTANTS} GATEWAY_REASON_CONSTANTS=${GATEWAY_REASON_CONSTANTS} \
+# GatewayClass conditions vary between standard and experimental
+GATEWAY_CLASS_CONDITION_CONSTANTS="GatewayClassConditionType=Accepted,SupportedVersion"
+GATEWAY_CLASS_REASON_CONSTANTS="GatewayClassConditionReason=Accepted,InvalidParameters,Pending,Unsupported,Waiting,SupportedVersion,UnsupportedVersion"
```

Comment:
```suggestion
GATEWAY_CLASS_CONDITION_CONSTANTS="${GATEWAY_CLASS_CONDITION_CONSTANTS},SupportedVersion"
GATEWAY_CLASS_REASON_CONSTANTS="${GATEWAY_CLASS_REASON_CONSTANTS},SupportedVersion,UnsupportedVersion"
```

An alternative idea, assuming `experimental` is always additive to `standard`.

Reply:
It is, in fact so yes let's go for that :+1: